src/superphot_plus/model/classifier.py

```diff
@@ -407,26 +280,22 @@ def evaluate_epoch(self, iterator):
 
         return epoch_loss / len(iterator), epoch_acc / len(iterator)
 
-    def test(self, test_features, test_classes, test_names, test_group_idxs, save_path):
+    def test(self, save_path):
         """Runs model over a group of test samples
 
         Parameters
         ----------
-        test_features : np.ndarray
-            The features array.
-        test_classes : np.ndarray
-            The classes array.
-        test_names : np.ndarray
-            The names array.
-        test_group_idxs : np.ndarray
-            The indices for each test set.
+        save_path
+            Where to store the probability results.
```

Comment:
It might be worth having `save_path` default to `None` and skipping the output in that case. This would allow you to skip a file write if you do not need it.